controllers/apps/apimanager_status_reconciler.go

```diff
@@ -224,3 +226,47 @@ func (s *APIManagerStatusReconciler) defaultRoutesReady() (bool, error) {
 
 	return allDefaultRoutesReady, nil
 }
+
+func (s *APIManagerStatusReconciler) reconcileHpaWarningMessages(conditions *common.Conditions, cr *appsv1alpha1.APIManager, client2 client.Client) {
```

Comment:
we can use s.Client() here instead.

Reply:
@austincunningham we don't use client2 now, can we remove it?

Reply:
good point